artemis-dto/src/main/java/org/apache/activemq/artemis/dto/XmlUtil.java

```diff
@@ -95,6 +95,7 @@ public static <T> T decode(Class<T> clazz,
       unmarshaller.setSchema(schema);
 
       Properties props = new Properties(System.getProperties());
+      props.putAll(System.getenv());
```

Comment:
This would create an ordering effect with the opposite precedence than I would typically expect, both from other configuration bits I've come across elsewhere, and also the other XMLUtil from Artemis. Worth flipping?